exercises/practice/strain/strain_spec.lua

```diff
@@ -0,0 +1,76 @@
+local keep = require 'strain'.keep
+local discard = require 'strain'.discard
+
+describe('strain', function()
+  local function always_true() return true end
+  local function always_false() return false end
+  local function is_even(x) return x % 2 == 0 end
+  local function is_odd(x) return x % 2 == 1 end
+  local function starts_with_z(s) return string.sub(s, 1, 1) == "z" end
+  local function contains_5(xs)
+    for _, x in ipairs(xs) do
+      if x == 5 then return true end
+    end
+    return false
+  end
+
+  describe('keep', function()
+    it('keep on empty list returns empty list', function()
+      assert.same({}, keep({}, always_true))
+    end)
+
+    it('keeps everything', function()
+      assert.same({1, 3, 5}, keep({1, 3, 5}, always_true))
```

Comment:
Can you format non-empty tables with a space after `{` and before `}` as in https://github.com/exercism/lua/blob/main/exercises/practice/etl/etl_spec.lua#L24?

Reply:
done